.vscode/tasks.json

```diff
@@ -0,0 +1,76 @@
+{
+  "version": "2.0.0",
+  "tasks": [
```

Comment:
Could you add a task or launch script for debugging without building?

Reply:
Not sure what you mean by debugging without building. Not that experienced with using vscode debugger as I'm mainly using Jetbrains myself (at least for debugging)

Reply:
File `.vcode/launch.json`:

```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "OAuth2 Proxy for Dex",
            "type": "go",
            "request": "launch",
            "mode": "auto",
            "program": "${workspaceFolder}",
            "args": [
                "--config", "contrib/local-environment/oauth2-proxy.cfg"
            ]
        },
        {
            "name": "OAuth2 Proxy for Keycloak",
            "type": "go",
            "request": "launch",
            "mode": "auto",
            "program": "${workspaceFolder}",
            "args": [
                "--config", "contrib/local-environment/oauth2-proxy-keycloak.cfg"
            ]
        },
        {
            "name": "OAuth2 Proxy with Alpha Config",
            "type": "go",
            "request": "launch",
            "mode": "auto",
            "program": "${workspaceFolder}",
            "args": [
                "--config", "contrib/local-environment/oauth2-proxy-alpha-config.cfg",
                "--alpha-config", "contrib/local-environment/oauth2-proxy-alpha-config.yaml"
            ]
        }
    ]
}
```